catalog/task/push-snapshot/0.10/tests/test-push-snapshot-fail-addgitshatag.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: test-push-snapshot-fail-addgitshatag
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: |
+    Run the push-snapshot task with addGitShaTag enabled but missing required data.
```

Comment:
what is missing? maybe add comment later on

Reply:
fixed